.mergify.yml

```diff
@@ -0,0 +1,23 @@
+pull_request_rules:
+  - name: automatic merge on CI success and review
+    conditions:
+      - "status-success=continuous-integration/drone/pr"
+      # Pull Request approval enforcement
+      - "#approved-reviews-by>=1"
+      - "#review-requested=0"
```

Comment:
What happens if a review is requested from someone and that person makes a review? Does that count as 0?

Reply:
I think so. https://doc.mergify.io/examples.html#id11